executor/distsql.go

```diff
@@ -118,11 +118,11 @@ func closeAll(objs ...Closeable) error {
 	return errors.Trace(err)
 }
 
-// timeZoneOffset returns the local time zone offset in seconds.
-func timeZoneOffset(ctx sessionctx.Context) int64 {
-	loc := ctx.GetSessionVars().GetTimeZone()
+// zone returns the local time zone offset in seconds.
+func zone(ctx sessionctx.Context) (string, int64) {
```

Comment:
This function should be the place to change `Local` to `System`.
`(h *rpcHandler) buildDAGExecutor` only affects `mocktikv`.

Reply:
Got it.

Reply:
I revisit codebase. I does not think `zone` is a good place to adjust this. Inside tidb, we still need 'Local' rather "System". I added the logic here and some ddl test failed.

Changing "Local" to "System" should only happens when we need push timezone name and timezone offset down to tikv side.